cmake/install_helpers.cmake

```
@@ -44,66 +44,63 @@ endfunction()
 function(ginkgo_install_library name)
     ginkgo_add_install_rpath("${name}" "${ARGN}")
 
-    if (WIN32 OR CYGWIN)
-        # dll is considered as runtime
-        install(TARGETS "${name}"
-            EXPORT Ginkgo
-            LIBRARY DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
-            ARCHIVE DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
-            RUNTIME DESTINATION "${CMAKE_INSTALL_FULL_BINDIR}"
-            )
-    else ()
-        # install .so and .a files
-        install(TARGETS "${name}"
-            EXPORT Ginkgo
-            LIBRARY DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
-            ARCHIVE DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
+    install(TARGETS "${name}"
+        EXPORT Ginkgo
+        LIBRARY
+        DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
+        COMPONENT Runtime
```

namelink can go into development component
```suggestion
        COMPONENT Runtime
        NAMELINK_COMPONENT Development
```

---

Great point! I missed that